<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPersonInfo</class>
 <widget class="QDialog" name="DialogPersonInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1666666</width>
    <height>166666</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>132</width>
        <height>467</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QWidget" name="infoWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>8</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelPhoto">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>仿宋</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="labelNo">
                <property name="font">
                 <font>
                  <family>仿宋</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>工号：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelName">
                <property name="font">
                 <font>
                  <family>仿宋</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>姓名：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelDepartment">
                <property name="font">
                 <font>
                  <family>仿宋</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>部门：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelPosition">
                <property name="font">
                 <font>
                  <family>仿宋</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>职位：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelRole">
                <property name="font">
                 <font>
                  <family>仿宋</family>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>角色：</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="employeeList">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>240</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>240</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="rightWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
